gcp-repo/metadata.product

```diff
@@ -4,7 +4,7 @@
   This product exists to (1) bring in any required features, and
   (2) add any additional metadata for the CT4E repository.
 -->
-<product uid="com.google.cloud.tools.eclipse.dist" version="1.1.0.qualifier" useFeatures="true" includeLaunchers="false">
```

Comment:
You need this change, though it doesn't hurt: it's not really visible unless people start examining the p2 metadata.